templates/politician/politician_list.html

```diff
@@ -51,22 +51,24 @@ <h1>Politicians</h1>
 
 <form name="state_code_form" method="get" action="{% url 'politician:politician_list' %}">
     {% csrf_token %}
+    <input type="hidden" name="politician_search" value="{{ politician_search|default_if_none:'' }}" />
+    <input type="hidden" name="show_all" value="{{ show_all|default_if_none:0 }}" />
+    <input type="hidden" name="exclude_politician_analysis_done" value="{{ exclude_politician_analysis_done|default_if_none:0 }}" />
```

Comment:
exclude_politician_analysis_done is already included below as a checkbox, and doesn't need to be included here as a hidden value.